### ISSUE TITLE
Porting to Plasma6

### ISSUE THREAD

**Issue by jsaenzh01** [MakG10/plasma-applet-server-status#11]
With Plasma6, Plasma 5 widget doesn't work. Have you think in upgrade it?

Thanks

**Comment by Cris70** [comment]
Yes please, I'd really like to have this widget in plasma6!! :-)

**Comment by Peter-Leh** [comment]
Please port your programme for Plasma6.  I like it a lot and really need it. Thank you!

**Comment by renyuneyun** [comment]
To whom still wondering: I have made a port to Plasma 6 https://github.com/renyuneyun/plasma-applet-server-status
You may also download it directly from Pling https://www.pling.com/p/2300078


I'm using it on my desk, so most (if not all) functions should be working.
But I do see some improvement points -- unfortunately I'm not experienced enough with Plasmoids or QML, so fixing them will take some time... Happy to take PRs (but bear with me for delays... quite busy with my job recently).
